staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -33,6 +34,12 @@ import (
 	fqrequest "k8s.io/apiserver/pkg/util/flowcontrol/request"
 	"k8s.io/apiserver/pkg/util/shufflesharding"
 	"k8s.io/klog/v2"
+
+	// The following hack is needed to work around a tooling deficiency.
+	// Packages imported only for test code are not included in vendor.
+	// See https://kubernetes.slack.com/archives/C0EG7JC6T/p1626985671458800?thread_ts=1626983387.450800&cid=C0EG7JC6T
+	// The need for this hack will be removed when we make queueset use an EventClock rather than a PassiveClock.
+	_ "k8s.io/utils/clock/testing"
```

Comment:
@MikeSpreitzer i can build https://github.com/kubernetes/kubernetes/pull/103829 on top of this PR so queueset uses `EventClock`, unless you are already working on it.

Reply:
Please go ahead, I was hoping you would.

Reply:
sounds good, once this PR merges, I will go ahead and rework https://github.com/kubernetes/kubernetes/pull/103829 to use `EventClock` for queuset implementation.